doc/sailfish-webview-popups-authpopupinterface.qdoc

```diff
@@ -25,6 +25,31 @@
   \sa UserPromptInterface, custompopups
 */
 
+/*!
+  \qmlproperty string AuthPopupInterface::messageKey
```

Comment:
is "key" the terminology used in gecko/mozilla?  In Qt we use "id" (e.g. translation id) rather than "key" but obviously we should be consistent with gecko terminology in this case.

Reply:
At least "key" is the term that's used in the files I've been touching. I've essentially been pushing calls to `getLocalizedString(key, formatArgs)` [like this](https://github.com/sailfishos/gecko-dev/blob/66249d15cd1213d67b0a44c9e6c4491260d114b8/toolkit/components/prompts/src/nsPrompter.js#L115) up into the QML layer.

Admittedly I didn't give it much thought before and I wouldn't mind using `id` instead if this would be clearer in the context?